community/server/src/docs/ops/powershell.asciidoc

```diff
@@ -44,11 +33,29 @@ Set-ExecutionPolicy -ExecutionPolicy RemoteSigned
 For more information see http://go.microsoft.com/fwlink/?LinkID=135[About execution policies].
 --
 
+[NOTE]
+--
+The powershell module will display a warning if it detects that you do not have administrative rights.
+--
+
+[[powershell-module-import]]
+== How do I import the module?
+
+The module file is located in the _bin_ directory of your Neo4j installation, i.e. where you unzipped the downloaded file.
+For example, if Neo4j was installed in _C:\Neo4j_ then the module would be imported like this:
+
+[source,powershell]
+----
+Import-Module C:\Neo4j\bin\Neo4j-Management.psd1
+----
+
+This will add the module to the current session.
+
 Once the module has been imported you can start an interactive console version of a Neo4j Server like this:
 
 [source,powershell]
 ----
-'C:\Neo4j' | Start-Neo4jServer -Console -Wait
+Invoke-Neo4j console
 ----
 
 To stop the server, issue `ctrl-c` in the console window that was created by the command.

```

Comment:
`ctrl-c` is written Ctrl-C on the Server Installation page. We don't have a style rule for writing key chords, but let's be consistent and write it Ctrl-C here also.